SUMMARY.md

```diff
@@ -35,6 +35,7 @@ This is the summary of user documentation.
       * [Customizing Ciphers](installation/ssl_tls/setting_up_ciphers.md)
       * [Change SSL Certificates](installation/ssl_tls/custom_server_certificate.md)
       * [End to end transport security](installation/ssl_tls/end_to_end_transport_security.md)
+      * [Configuring HSTS Header](installation/ssl_tls/configuring_hsts_header.md)
```

Comment:
Not sure if this can be part of SSL/TLS?

Reply:
Merged the PR after discussing it with @akshaydewan. A user has to enable `SSL\TLS` in order to make use of HSTS. When TLS is not enabled client will ignore the HSTS header.